### PR TITLE
Added Plus models and specified iPad Pro models

### PR DESCRIPTION
**Pull request by ipadkid358**
Added Plus models, iPad Pro models, and fixed iPod Touch name

### CODE REVIEW

**Comment by nullpixel1** [comment]
There's no need for plus models, they are identical to their counterparts.

**Comment by DennisBlack** [comment]
It's more clear this way, I like it

File: README.md [hunk 1]
```diff
@@ -24,13 +26,15 @@ In the near future, the jailbreak will support the following devices:
...
 | iPad Air 2| iOS 10.0.0 -> iOS 10.2 |
 | iPad Mini 3| iOS 10.0.0 -> iOS 10.2 |
-| iPod touch (6G)  | iOS 10.0.0 -> iOS 10.2 |
+| iPod Touch (6G)  | iOS 10.0.0 -> iOS 10.2 |
 | iPad Mini 4 | iOS 10.0.0 -> iOS 10.2 |
 | iPhone 7  | iOS 10.0.0 -> iOS 10.1.1 |
+| iPhone 7 Plus | iOS 10.0.0 -> iOS 10.1.1 |
 
-**Note, the iPhone 7 is only supported till iOS 10.1.1**
+**Note, the iPhone 7 and 7 Plus is only supported till iOS 10.1.1**
```
Comment (AppleBetas) [inline README.md]: ~~is~~ are

Comment (MelkyPie) [inline README.md]: Shouldn't it actually be: "will be supported" , as they are not supported yet?

Comment (meyer9) [inline README.md]: until

**Comment by tangalbert919** [comment]
The plus models are not identical to their counterparts due to a different model number, kernel, and their added ability to rotate on the SpringBoard.
They are, however, extremely similar counterparts, but that doesn't make it the same. This is a common misperception that can throw off programmers hard. Plus, it will make people stop asking about whether or not Plus models are supported.

**Comment by toniqyteza** [comment]
you forgot to add iPhone 5s

**Comment by ipadkid358** [comment]
@toniqyteza no I didn't

**Comment by ipadkid358** [comment]
@AppleBetas, thank you. Updated

**Comment by kpwn** [comment]
> The plus models are not identical to their counterparts due to a different model number, kernel, and their added ability to rotate on the SpringBoard.
They are, however, extremely similar counterparts, but that doesn't make it the same. This is a common misperception that can throw off programmers hard. Plus, it will make people stop asking about whether or not Plus models are supported.

Wrong.

**Pull request closed without merging**